tests/cmd/check-spelling/kata-dictionary.dic

```diff
@@ -331,6 +331,7 @@ rootfs/AB
 runV/B
 runc/B
 runtime/AB
+runtimeClass/
```

Comment:
You can skip the `/` when no grammar is specified (would be consistent with other simple words...)

Reply:
fixed